README.md

```diff
@@ -552,6 +552,23 @@ oneview_unmanaged_device 'UnmanagedDevice1' do
 end
 ```
 
+### oneview_uplink_set
+
+Uplink set resource for HPE OneView
+
+```ruby
+oneview_unmanaged_device 'UnmanagedDevice1' do
+  client <my_client>
+  data <data>
+  native_network # The name of the network that is designated as the native network
+  fc_networks # Array of assigned FC networks
+  fcoe_networks # Array of assigned FCoE networks
+  networks # Array of assigned Ethernet networks
```

Comment:
Please be specific as to what these arrays should include. For example, these are arrays of network **names**, which are strings.

Also, if these properties are optional, please say so.